.gitignore

```diff
@@ -60,4 +60,6 @@ venv/
 .vscode/
 
 # Logs
-**.log
\ No newline at end of file
+**.log
+
+.env
```

Comment:
shouldn't you use .env.local anyways for next js?

Reply:
Yes, corrected now